src/main/java/org/jabref/model/entry/Date.java

```diff
@@ -60,7 +61,8 @@ public Date(TemporalAccessor date) {
 
         for (String formatString : formatStrings) {
             try {
-                TemporalAccessor parsedDate = DateTimeFormatter.ofPattern(formatString).parse(dateString);
+                // Locale is required for parsing month names correctly. Currently this expects the month names to be in english
+                TemporalAccessor parsedDate = DateTimeFormatter.ofPattern(formatString).withLocale(Locale.UK).parse(dateString);
```

Comment:
Locale.UK maybe provoke a wtf-moment, even though it has nothing to do with the parser here. Maybe it's better to choose Locale.US (silly US date format: MM-DD-YYYY, plain normal UK format: DD-MM-YYYY)?

Reply:
Yeah you are right! I just added it for parsing plaintext months that otherwise did not work and forgot that it also might influence the format ^^